emoji-ui/src/store/actions/Pagination.js

```diff
@@ -2,7 +2,7 @@ import ActionTypes from '../ActionTypes';
 
 function setCurrentPage(pageNum) {
   return {
-    payload: pageNum,
+    payload: parseInt(pageNum || 0),
```

Comment:
pageNum 0 is an invalid number so don't use it as a default.

Reply:
you can also leave a comment saying "callers must sanitize pageNum" and ignore it here to be faster. it's easier to do in the component.

Reply:
ok!